index.html

```diff
@@ -5,7 +5,19 @@
   <title>Quantified Self - Diary</title>
 </head>
 <body>
-
+  <div class="left">
+  <h2 class="left"></h2>
+  </div>
+  <div class="right">
+  <h2 class="right"></h2>
+  </div>
+  <div class="left1">
+  <h2 class="left1"></h2>
+  </div>
+  <div class="right1">
+  <h2 class="right1"></h2>
+  </div>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Thanks for the heads up on using the script from line 20!! I forgot we'd need to require jQuery in JS apps. I think if we wanted to, we could probably NPM install it and require it in index: 

npm install jQuery
var $ = require('jQuery');